frontend/src/ProfilePage/Profile.jsx

```diff
@@ -320,8 +344,24 @@ function Profile() {
                 ) : (
                     <div></div>
                 )
+                }
+                {
+                    Object.keys(clickedBoxDetails).length > 0 ? (
+                        <div className='ClickedDetails'>
+                            <p>25th Percentile: {clickedBoxDetails.quantiles[0].toFixed(2)} </p>
```

Comment:
nit: is quantiles guaranteed to have 3 elements? and range guaranteed to have 2 elements? If not, we should do a check here before using them

Reply:
Yes, the number of elements of each is fixed at 3 and 2 since they're created like so:`quantiles: [bottomQuantile, medianQuantile, topQuantile], range: [bottomOutlierRange, topQuantileRange],`.